torch/_dynamo/eval_frame.py

```diff
@@ -813,6 +886,10 @@ def graph_with_interpreter(*args):
         graph,
     ).transform()
 
+    new_graph = AddRuntimeAssertsInInputConstraint(
+        new_graph,
+    ).transform()
```

Comment:
Hmmph. At some point we should probably start putting these in their own module. It's also annoying that these are inner class definitions as you cannot easily tell if they closed over things or not.